app/views/global_search/_products.html.haml

```diff
@@ -1,22 +1,22 @@
 %h3= Product.model_name.human(count: 2)
 
-%table.table.table-striped.table-hover.js--responsive_table
+%table.table.table-striped.table-hover.js--responsive_table.table-product-results
   %thead
     %tr
-      %th= Product.model_name.human
-      %th= Product.human_attribute_name("type")
-      %th= Facility.model_name.human
-      %th= Product.human_attribute_name("description")
+      %th.name= Product.model_name.human
```

Comment:
Should the headers have `.table-product-results__name` as well?

Reply:
Yeah! 3a29672ec does this (it doesn’t matter because the width will be dictated by the `td`s anyway, but it’s better than having unused classes for sure 😄)